src/System.Drawing.Common/src/System/Drawing/Bitmap.cs

```diff
@@ -12,6 +12,7 @@
 namespace System.Drawing
 {
     [Serializable]
```

Comment:
Did you do a search for all of these that were added?

Reply:
Yes, we're not mising any.